src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -182,20 +179,20 @@ function Test-GetAzureStorageAccount
 
         Retry-IfException { $global:sto = Get-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname; }
         $stotype = 'StandardGRS';
-        Assert-AreEqual $sto.StorageAccountName $stoname;
-        Assert-AreEqual $sto.Sku.Name $stotype;
-        Assert-AreEqual $sto.Location $loc;
-        Assert-AreEqual $sto.Kind $kind;
+        Assert-AreEqual $stoname $sto.StorageAccountName;
+        Assert-AreEqual $stotype $sto.Sku.Name;
+        Assert-AreEqual $loc $sto.Location;
+        Assert-AreEqual $kind $sto.Kind;
         # TODO: Temporary removal because of https://github.com/Azure/azure-powershell/issues/5230
-        #Assert-AreEqual $sto.EnableHttpsTrafficOnly $false;
+        #Assert-AreEqual $false $sto.EnableHttpsTrafficOnly;
```

Comment:
Can you reenable this assert across the board?

Reply:
Since the related server bug is fixed, I have re-enable it.